x-pack/plugins/security_solution/public/common/components/event_details/insights/insights.tsx

```diff
@@ -4,6 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+/* eslint-disable complexity */
```

Comment:
nit: if this `eslint-disable complexity` is removed, the entire file is highlighted, but temporarily changing just one randomly-chosen block of conditionals:

```ts
    const canShowAtLeastOneInsight =
      hasCasesReadPermissions ||
      hasProcessEntityInfo ||
      hasSourceEventInfo ||
      hasProcessSessionInfo;
```

to

```ts
    const canShowAtLeastOneInsight = true;
```

was enough to remove the highlighting for the entire file. Consider extracting this block of code into a helper function in a separate file and removing the `eslint-disable complexity`.

Reply:
Ah yes, nice one. I added a type guard here https://github.com/elastic/kibana/pull/138419/commits/24131f7be52465b2863577638e32d14dfa21443b 🎉